lightning/src/routing/gossip.rs

```diff
@@ -2800,6 +2939,71 @@ mod tests {
 		assert_eq!(format_bytes_alias(b"\xFFI <heart>\0LDK!"), "\u{FFFD}I <heart>");
 		assert_eq!(format_bytes_alias(b"\xFFI <heart>\tLDK!"), "\u{FFFD}I <heart>\u{FFFD}LDK!");
 	}
+
+	#[test]
+	fn written_channel_info_is_readable() {
```

Comment:
While this test was nice for debuggin purposes, it's actually quite redundant now that the deserialization works again. I therefore intend to remove it before this lands, but let me know if you think it adds value and should stay.

Reply:
On the contrary I now extended the test with https://github.com/lightningdevkit/rust-lightning/pull/1519/commits/331996b4ba8b709a5750ee2c8479f8b9169bb40c.